assets/js/admin/facebook-for-woocommerce-settings-sync.js

```diff
@@ -147,4 +147,137 @@ jQuery( document ).ready( function( $ ) {
 		}
 	} );
 
+	// mark as in-progress if syncing when the page is loaded
+	if ( facebook_for_woocommerce_settings_sync.sync_in_progress ) {
+		syncInProgress();
+	}
+
+	// handle the sync button click
+	$( '#woocommerce-facebook-settings-sync-products' ).click( function( event ) {
+
+		event.preventDefault();
+
+		if ( confirm( facebook_for_woocommerce_settings_sync.i18n.confirm_sync ) ) {
+
+			syncInProgress();
+
+			$.post( facebook_for_woocommerce_settings_sync.ajax_url, {
+				action: 'wc_facebook_sync_products',
+				nonce:  facebook_for_woocommerce_settings_sync.sync_products_nonce,
+			}, function ( response ) {
+
+				console.log( response );
```

Comment:
Do we want to always log the response?

Reply:
yeah I think that's fine since we're in the admin context